Packs/CybleEvents/IncidentFields/incident_cybleeventsname.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
@cyble-dev Please use the common field "Event Names" instead.

Reply:
the type of common field is different so will be using custom field here